.gitignore

```diff
@@ -9,6 +9,8 @@ yarn-error.log
 /fixtures/test
 /fixtures/my-remix-app
 /fixtures/deno-app
+/playwright-report
+/test-results
```

Comment:
When does this folder show up? I'm not getting it locally.

Reply:
it'll only show up if you have `CI=1` it looks like due to https://github.com/remix-run/remix/pull/2634/files#diff-79d1eafd16336460c9185ab9aa820bddafd171a2f65918d8c06968a8e8b69bb4R96-R97, but we probably don't need the trace files, i just let them initially as `npm init playwright` gave it to me

Reply:
Cool. Let's leave it in then 👍